guides/micronaut-oracle-cloud-streaming/chess-game/java/src/main/java/example/micronaut/chess/GameController.java

```diff
@@ -0,0 +1,81 @@
+package example.micronaut.chess;
+
+import example.micronaut.chess.dto.GameDTO;
+import example.micronaut.chess.dto.GameStateDTO;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.PathVariable;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.reactivex.Single;
+
+import java.util.UUID;
+
+import static io.micronaut.http.MediaType.APPLICATION_FORM_URLENCODED;
+import static io.micronaut.http.MediaType.TEXT_PLAIN;
+
+/**
+ * Endpoints called by the front end using Ajax.
+ */
+@Controller("/game")
+@ExecuteOn(TaskExecutors.IO) // <1>
+class GameController {
+
+    private final GameReporter gameReporter;
+
+    GameController(GameReporter gameReporter) {
+        this.gameReporter = gameReporter;
+    }
+
+    /**
+     * Start a new game.
+     *
+     * @param b black's name
+     * @param w white's name
+     * @return the id of the game
+     */
+    @Post(value = "/start", consumes = APPLICATION_FORM_URLENCODED, produces = TEXT_PLAIN) // <2>
+    Single<String> start(String b, String w) { // <3>
```

Comment:
`@SingleResult Publisher<String>` instead of `Single<String>`.

Reply:
I've created a discussion outside of the PR to come up with a decision: https://github.com/micronaut-projects/micronaut-guides/discussions/393